python/lsst/obs/sdss/scaleSdssZeroPoint.py

```diff
@@ -74,11 +79,11 @@ def getInterpImage(self, bbox):
         """
 
         npoints = len(self._xList)
-        #sort by X coordinate
+        # sort by X coordinate
         if npoints < 1:
             raise RuntimeError("Cannot create scaling image. Found no fluxMag0s to interpolate")
 
-        x, z = zip(*sorted(zip(self._xList, self._scaleList)))
+        x, z = list(zip(*sorted(zip(self._xList, self._scaleList))))
```

Comment:
I think this is one of those things with no great pythonic solution. If you have a better one I'd be happy to implement it, but unless I'm using numpy arrays I usually use this technique.